x-pack/plugin/profiler/src/internalClusterTest/java/org/elasticsearch/xpack/profiler/ProfilingTestCase.java

```diff
@@ -71,16 +83,31 @@ private void indexDoc(String index, String id, Map<String, Object> source) {
      */
     protected abstract boolean useOnlyAllEvents();
 
+    protected void waitForIndices() throws Exception {
+        // block until the templates are installed
+        assertBusy(() -> {
+            ClusterState state = client().admin().cluster().prepareState().get().getState();
+            // simple check whether at least one of the indices is present
```

Comment:
```suggestion
            // simple check whether all the indices is present
```

If I am not mistaken this is checking if all the indices have been created, right?

Reply:
Addressed in https://github.com/elastic/elasticsearch/pull/95666/commits/7dc6f1ca856384d427785431685813a769ab0c8b (I've removed the comment entirely as I think the code is clear enough as it is done now).